sstables/mx/writer.cc

```diff
@@ -1385,6 +1386,8 @@ void writer::write_clustered(const rt_marker& marker, uint64_t prev_row_size) {
     write_vint(*_data_writer, _tmp_bufs.size());
     flush_tmp_bufs();
     _collector.update_min_max_components(marker.position());
+    ++_c_stats.range_tombstones_count;
+    collect_row_stats(_data_writer->offset() - current_pos, &marker.clustering);
```

Comment:
Why this new call?

Reply:
To count range tombstone markers also as rows since they are clustering elements and are processed along with rows in the cache.
Ideally we may want to count them separately but I didn't want to make the threshold logic even more complex than it already is.